src/components/search-form/search-form.js

```diff
@@ -90,6 +119,74 @@ class SearchForm extends Component {
     return null;
   };
 
+  getSortedDataOptions = () => {
+    const { tagsModel = [] } = this.props;
+    const dataOptions = tagsModel.map(tag => {
+      return {
+        value: tag.label.toLowerCase(),
+        label: tag.label.toLowerCase(),
```

Comment:
Could you move `tag.label.toLowerCase()` to a variable as a duplication?

Reply:
all set!